Documentation/gettingstarted/encryption.rst

```diff
@@ -71,15 +77,38 @@ to GKE, with VXLAN tunneling:
 .. parsed-literal::
 
     helm install cilium |CHART_RELEASE| \\
-      --namespace cilium \
-      --set nodeinit.enabled=true \
-      --set nodeinit.reconfigureKubelet=true \
-      --set nodeinit.removeCbrBridge=true \
-      --set cni.binPath=/home/kubernetes/bin \
-      --set tunnel=vxlan \
-      --set encryption.enabled=true \
+      --namespace kube-system \\
```

Comment:
The GKE example should be kept in `cilium` namespace. GKE deletes DaemonSets that are installed in the kube-system namespace.

Reply:
> GKE deletes DaemonSets that are installed in the kube-system namespace.

Did we recheck this recently? I can know deploy in the `kube-system` namespace without issues.

Reply:
Is this related? https://github.com/kubernetes/kubernetes/issues/51376

Doesn't seem to be isolated to GKE, and it worked fine in my tests as well. :thinking:  There have been some commits to `cilium/cilium` that refer to the above issue, PTAL.

Looking at the codebase, we use `--namespace cilium` in a couple of guides including OpenShift, Azure and EKS. I proposed this change because it doesn't align with the `Secret` installation command above.

So, do we stick to `cilium`?

Reply:
>So, do we stick to cilium?

yes, for now.

Reply:
As discussed offline, all examples in the docs will be changed to `kube-system`.